Dockerfile

```diff
@@ -0,0 +1,20 @@
+ARG PYTHON_VERSION=3.8
+
+FROM python:${PYTHON_VERSION}
+
+ENV ENV="Local"
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+COPY requirements.txt .
```

Comment:
Consider using a specific directory for copying `requirements.txt` to avoid potential conflicts with other files. For example, you could create a `requirements` directory and copy it there.